admin/proc-using-azure-auth-provider.adoc

```diff
@@ -0,0 +1,141 @@
+[id='proc-using-azure-auth-provider_{context}']
+= Using Microsoft Azure as an authentication provider in {product}
+
+The `core-plugin-api` package in {product-short} comes integrated with Microsoft Azure authentication provider, authenticating signing in using Azure OAuth.
```

Comment:
Consider _Red Hat Developer Hub uses a built-in Microsoft Azure authentication provider to authenticate users in Azure._ for consistency with the GitHub section

Reply:
The SME recommended removing the _`core-plfuin-api` package_ part from those docs

Reply:
I don't see any comment from SME. Could you please share the link?